.github/workflows/chart-release.yml

```diff
@@ -14,20 +15,19 @@ jobs:
         uses: actions/checkout@v4
         with:
           fetch-depth: 0
-
-      - name: Configure Git
+          submodules: recursive
+          
+      - name: Block publish if version is a develop pre-release
         run: |
-          git config user.name "$GITHUB_ACTOR"
-          git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
-
-      - name: Set up Helm
-        uses: azure/setup-helm@v4
-        with:
-          version: v3.14.3
+          VERSION=$(yq '.version' charts/Chart.yaml)
+          if [[ "$VERSION" == *-develop* ]]; then
+            echo "Chart version is a develop prerelease: $VERSION"
+            echo "Aborting chart release on a release branch"
+            exit 1
```

Comment:
This will make commit show errors on github, instead please expose var and then skip other jobs in case of "skip"

Reply:
Fixed.